lib/common/transforms.js

```diff
@@ -287,6 +287,53 @@ module.exports = {
     }
   },
 
+    /**
+   * Transforms the value into an HSL string or HSLA if alpha is present. Better browser support than color/hsl
+   *
+   * ```js
+   * // Matches: prop.attributes.category === 'color'
+   * // Returns:
+   * "hsl(174, 100%, 29%)"
+   * "hsl(174, 100%, 29%, .5)"
+   * ```
+   *
+   * @memberof Transforms
+   */
+  'color/hsla': {
+    type: 'value',
+    matcher: isColor,
+    transformer: function (prop) {
+      return Color(prop.value).toHslString();
+    }
+  },
+
+    /**
+   * Transforms the value into an HSL string, using fourth argument if alpha is present.
+   *
+   * ```js
+   * // Matches: prop.attributes.category === 'color'
+   * // Returns:
+   * "hsl(174 100% 29%)"
+   * "hsl(174 100% 29% / .5)"
```

Comment:
This might be a dumb question, what is the difference in the formatting of this output versus the 'color/hsla' transform? Do both work and is one preferable over the other? Both transforms seem to handle alpha and both output `hsl()`, do we need both?